app/src/main/java/com/sun/makeupwindow/data/ProductDataSource.kt

```diff
@@ -6,13 +6,14 @@ import com.sun.makeupwindow.data.source.utils.OnDataLoadCallback
 interface ProductDataSource {
     interface Local{
         fun getProducts(callback: OnDataLoadCallback<List<Product>>)
-        fun getItemProduct(idproduct: Int, callback: OnDataLoadCallback<Product>)
+        fun getProductIdAfterLast(callback: OnDataLoadCallback<Int>)
```

Comment:
```suggestion
        fun getLastId(callback: OnDataLoadCallback<Int>)
```
đổi nên những nơi khác nữa. đặt tên sao cho vừa ngắn gọn lại vừa đủ nghĩa